recipes/linecache2/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{%set name = "linecache2" %}
+{%set version = "1.0.0" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "4b26ff4e7110db76eeb6f5a7b64a82623839d595c2038eeda662f2a2db78e97c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+  patches:
+    - inspect_fodder2.patch  # [not py3k]
+    - test_linecache.patch  # [not py3k]
+    # inspect_fodder2 kills the build when byte-compiling on py 2.x
```

Comment:
I believe this is one of those things that would not happen if we were using `pip` to install it. This is the 3rd package in conda-forge that needs this workaround... Maybe it is time to re-visit https://github.com/conda-forge/staged-recipes/issues/528